.buildkite/pipeline.yml

```diff
@@ -0,0 +1,4 @@
+steps:
```

Comment:
The pr pipeline didn't seem to run. Could it be the filename needs to be `pr.yml`

Reply:
odd let me try that